drag_and_drop_v2/utils.py

```diff
@@ -5,3 +5,24 @@
 # Make '_' a no-op so we can scrape strings
 def _(text):
     return text
+
+
+def ngettext(text_singular, text_plural, n):
+    if n == 1:
+        return text_singular
+    else:
+        return text_plural
+
+
+class FeedbackMessages(object):
+    FINAL_ATTEMPT_TPL = _('Final attempt was used, highest score is {score}')
+    MISPLACED_ITEMS_RETURNED = _('Misplaced items were returned to item bank.')
+
+    CORRECTLY_PLACED_SINGULAR_TPL = _('Correctly placed {correct_count} item.')
+    CORRECTLY_PLACED_PLURAL_TPL = _('Correctly placed {correct_count} items.')
```

Comment:
@mtyaka Also, what about [all this stuff](https://github.com/edx-solutions/xblock-drag-and-drop-v2/blob/master/drag_and_drop_v2/drag_and_drop_v2.py#L38-L134)? Looks like it is also translated too early, as strings are not updated even when I switch to other locale in studio: 

![image](https://cloud.githubusercontent.com/assets/3330048/17402374/9d2cfd86-5a5b-11e6-8ce7-40e48d27aa5a.png)

Note save and cancel buttons have changed captions, while all XBlock fields have not.